homeassistant/components/transmission/config_flow.py

```diff
@@ -59,18 +59,14 @@ async def async_step_user(self, user_input=None):
                 if (
                     entry.data[CONF_HOST] == user_input[CONF_HOST]
                     and entry.data[CONF_PORT] == user_input[CONF_PORT]
-                ):
-                    return self.async_abort(reason="already_configured")
-                if entry.data[CONF_NAME] == user_input[CONF_NAME]:
-                    errors[CONF_NAME] = "name_exists"
-                    break
-
+                ) or (entry.data[CONF_NAME] == user_input[CONF_NAME]):
+                    return self.async_abort(reason="already_configured_device")
```

Comment:
Please use `already_configured`. 

```suggestion
                    return self.async_abort(reason="already_configured")
```

Reply:
Done. Would you tell me why not matching the key is better here?

Reply:
`already_configured` is also used by `abort_if_unique_id_configured` check. So if we ever start using that check, it would need `already_configured` anyway, so it's good to already start using that.